photon-ml/src/integTest/scala/com/linkedin/photon/ml/cli/game/training/DriverTest.scala

```diff
@@ -123,7 +130,7 @@ class DriverTest extends SparkTestUtils with TestTemplateWithTmpDir {
   def testRandomEffectsWithIntercept(): Unit = sparkTest("testRandomEffectsWithIntercept", useKryo = true) {
     val outputDir = s"$getTmpDir/randomEffects"
     // This is a baseline RMSE capture from an assumed-correct implementation on 4/14/2016
-    val errorThreshold = 2.2
+    val errorThreshold = 2.34
```

Comment:
The test was previously using the wrong dataset to compute validation error. When I corrected it, the value changed. Need to revisit these tests, since they're not really giving much more signal than "yes, this still produces predictions in a somewhat reasonable range of reals".